src/main/java/at/hannibal2/skyhanni/data/PetAPI.kt

```diff
@@ -13,19 +47,128 @@ object PetAPI {
         "Pets(?: \\(\\d+/\\d+\\) )?",
     )
 
+    var pet: PetData? = null
+        private set
+    var inPetMenu = false
+        private set
+
+    private var xpLeveling: List<Int> = listOf()
+    private var petsDataNEU: Map<String, NEUPetData>? = null
+
+    private val nameToFakeInternalNameCache = mutableMapOf<String, String>()
+    private val nameToInternalNameCache = mutableMapOf<String, NEUInternalName>()
```

Comment:
what data is getting saved here? why cant this be dealt witht the exisitng caches in ItemUtils?